apiserver/common/storagecommon/blockdevices.go

```diff
@@ -32,6 +32,7 @@ func MatchingBlockDevice(
 	blockDevices []state.BlockDeviceInfo,
 	volumeInfo state.VolumeInfo,
 	attachmentInfo state.VolumeAttachmentInfo,
+	planBlockInfo state.BlockDeviceInfo,
```

Comment:
This method has no pre-existing tests, but can we get the ball rolling with one that tests the added path?